src/cmd/dashboard/monitor.js

```diff
@@ -287,97 +343,41 @@ class Monitor {
     this.screen.append(this.wrapper);
   }
 
-  layoutCmd() {
-    this.consoleBox = blessed.box({
-      label: __('Console'),
-      tags: true,
-      padding: 0,
-      width: '100%',
-      height: '6%',
-      left: '0%',
-      top: '95%',
-      border: {
-        type: 'line'
-      },
-      style: {
-        fg: 'black',
-        border: {
-          fg: this.color
-        }
-      }
-    });
-
-    this.input = blessed.textbox({
-      parent: this.consoleBox,
-      name: 'input',
-      input: true,
-      keys: false,
-      top: 0,
-      left: 1,
-      height: '50%',
-      width: '100%-2',
-      inputOnFocus: true,
-      style: {
-        fg: 'green',
-        bg: 'black',
-        focus: {
-          bg: 'black',
-          fg: 'green'
+  layoutTerminal() {
+      this.terminal = blessed.terminal({
+        parent: this.screen,
+        cursor: 'block',
+        cursorBlink: true,
+        padding: 0,
+        width: '100%',
+        height: 3,
+        left: 0,
+        top: '100%-3',
+        border: 'line',
+        style: {
+            fg: 'default',
+            bg: 'default',
+            focus: {
+                border: {
+                    fg: 'green'
+                }
+            }
+        },
+        scrollable: false,
+        handler: (data) => {
+          this.terminalReadableStream.push(data);
```

Comment:
Some of those layout changes affect the "Available Services" box, which now bleeds into the Logs section